test/CMakeLists.txt

```diff
@@ -295,7 +295,12 @@ celeritas_add_test(celeritas/global/ActionManager.test.cc)
 celeritas_add_test(celeritas/global/AlongStep.test.cc
   ${_optional_geant4_env})
 celeritas_add_test(celeritas/global/Stepper.test.cc
-  GPU ${_needs_geant4})
+  GPU ${_needs_geant4}
+  FILTER
+    # NOTE: these can be run in the same invocation once Geant4 reload is fixed
+    "TestEm3*"
+    "TestEm15FieldTest.*"
```

Comment:
Not sure this is working as intended?

Reply:
I think it is? When I run `ctest`, instead of one crashing test I get two separate calls to the test executable (with gtest-filter args to enable one component). What are you seeing?

Reply:
Ah nope you are right!